crates/sui-json-rpc-types/src/sui_object.rs

```diff
@@ -923,6 +925,12 @@ pub enum SuiObjectDataFilter {
         #[serde_as(as = "DisplayFromStr")]
         StructTag,
     ),
+    Owner(SuiAddress),
+    ObjectOwner(ObjectID),
+    ObjectId(ObjectID),
+    // allow querying for multiple object ids
+    ObjectIds(Vec<ObjectID>),
+    Version(u64),
```

Comment:
what is this for? from literal meaning it means all objects of certain version, but presumably objects of the same version are not relevant?

Reply:
Unsure @666lcz ? I'm assuming it might be useful to get object history.